marketplace/deployer_util/config_helper.py

```diff
@@ -441,7 +694,26 @@ def base64_encoded(self):
 
 
 class SchemaXReportingSecret:
-  """Wrapper class providing convenient access to REPORTING_SECRET property."""
+  """Accesses REPORTING_SECRET property."""
 
   def __init__(self, dictionary):
     pass
+
+
+def _get_or_invalid(dictionary, key, error_msg):
+  if key not in dictionary:
+    raise InvalidSchema(error_msg)
+  return dictionary[key]
+
+
+def _apply_or_none(dictionary, key, apply_fn):
```

Comment:
ditto: could be something like: apply_if_exists or apply_if_not_none

Reply:
changed to `_maybe_get_and_apply`